### DISCUSSION TITLE
DiaNN 1.8.1 for SILAC data does not return non-unimod identifiers

### DISCUSSION THREAD

**Discussion by pkohvaei** [vdemichev/DiaNN#564] [Q&A] [answered]
Dear all, 

DiaNN 1.8.1 is installed on Windows server 2016.
This is the command line (I write the relevant arguments here):
```
DiaNN.exe 
--f file_name 
--lib 2channel_speclib.tsv
--fasta fasta_file 
--met-excision 
--cut K*,R* 
--missed-cleavages 1 
--min-pep-len 7 
--max-pep-len 30 
--min-pr-mz 300 
--max-pr-mz 1800 
--unimod4 
--peak-center 
--no-ifs-removal 
--qvalue 0.01 
--reannotate 
--min-pr-charge 1 
--max-pr-charge 4 
--window 6 
--mass-acc 10 
--mass-acc-ms1 5 
--rt-profiling 
--smart-profiling 
--fixed-mod SILAC,0.0,KR,label 
--channels "SILAC,L,KR,0:0;SILAC,H,KR,8.014199:10.008269" 
--peak-translation 
--original-mods 
--ms1-base-profile
```

We expect to see the non-unimod presentation in the report.tsv, e.g.:             
```
[979] "AAAAC(UniMod:4)LDK(SILAC-K-H)"              
[980] "AAAAC(UniMod:4)LDK(SILAC-K-H)"              
[981] "AAAAC(UniMod:4)LDK(SILAC-K-H)"              
[982] "AAAAC(UniMod:4)LDK(SILAC-K-H)"              
[983] "AAAAC(UniMod:4)LDK(SILAC-K-H)"              
[984] "AAAAC(UniMod:4)LDK(SILAC-K-H)"  
```
but we get the unimod presentation instead (which would be the same output if we run it as a labelfree experiment, without any SILAC settings:
```
[981] "AALLSGPPGVGK(UniMod:259)"                                                        
[982] "AALLSGPPGVGK"                                                                    
[983] "AALLSPGDPAIWAGLM(UniMod:35)AAC(UniMod:4)HADDK(UniMod:259)"                       
[984] "AALLSPGDPAIWAGLM(UniMod:35)AAC(UniMod:4)HADDK"         
```                          

Is there an explanation for tis behavior? What am I missing?

Thank you for your help,
Parastou

**Comment by vdemichev** [comment, marked as answer]
Hi Parastou,

The above command will only work in the library contains '(SILAC)' after each K or R. Is this the case? 

Best,
Vadim

**Reply by pkohvaei** [reply]
Hi Vadim,

That must be the case, the library we use here has been generated by DiaNN1.7.10 and for sure does not contain "SILAC" notation.
We will try with a different library.

Thank you very much!

**Reply by vdemichev** [reply]
--lib-fixed-mod will add SILAC labels to the library. Also, can send the details on how to configure SILAC in general in DIA-NN via email

**Reply by pkohvaei** [reply]
Thanks Vadim!

Yes, I ran the same command - with new speclib - several times until I noticed "-lib-fixed-mod SILAC" is also needed to see this:
```
AAAAASAAGPGGLVAGK(SILAC-K-L)
AAAAAWEEPSSGNGTAR(SILAC-R-L)
...
```
That would be great if you send the details on how to configure SILAC in general in DIA-NN via email.
